### PR TITLE
Edited Introduction for Simplicity and brevity

### PR DESCRIPTION
**Pull request by codenihal**
Edited the Introduction for simplicity and brevity.

### CODE REVIEW

File: 01-introduction/01-introduction.md [hunk 1]
```diff
@@ -4,11 +4,11 @@
 
 Lotus is a Ruby MVC web framework comprised of many micro-libraries. It has a
 simple, stable API, a minimal DSL, and prioritises the use of plain objects over
-magical, over-complicated classes with too much responsibility.
+magical, cluttered classes with too much responsibility.
```
Comment (tomkadwill) [inline 01-introduction/01-introduction.md]: I prefer 'over-complicated' in this context

**Comment by tomkadwill** [comment]
I've made one recommendation, other than that I think its good to merge

**Comment by codenihal** [comment]
Made the change. :)

**Comment by tomkadwill** [comment]
:+1:

**Comment by jodosha** [comment]
@codenihal @tomkadwill Thank you for this PR. Now that we have the new website, I'm writing Guides (yes, Guides! :confetti_ball: ) at: https://github.com/lotus/lotus.github.io Would you mind to open PRs over there instead? We need proof reading by english native speakers.

**They are NOT published yet. Planning to do in the near future.** Please don't spread the word yet ;)

**Comment by tomkadwill** [comment]
@jodosha Sounds great! will do

**Comment by codenihal** [comment]
Of course, I'm proofreading already. Have to say that I'm really excited for Lotus. If we can make it a bit more usable, and provide better documentation. Lotus will lead the pack :+1:

**Pull request merged**